cmd/buckler/main.go

```diff
@@ -842,9 +855,19 @@ func main() {
 			}
 			opts.Rekey.NewMaster = a
 		}
-		err := c.Rekey(opts.Rekey.OldMaster, opts.Rekey.NewMaster)
+		disasterKey, err := c.Rekey(opts.Rekey.OldMaster, opts.Rekey.NewMaster, opts.Rekey.RekeyDR)
 		bail(err)
 
+		if disasterKey != "" {
+			fmt.Printf("@R{BELOW IS YOUR DISASTER KEY FOR RECOVERING FIXED-KEY BACKUPS.}\n")
+			fmt.Printf("@R{SAVE THIS IN A SECURE LOCATION.}\n")
+			fmt.Printf("----------------------------------------------------------------\n")
+			fmt.Printf("@Y{" + c.SplitKey(disasterKey, 64) + "}")
+			fmt.Printf("\n----------------------------------------------------------------\n")
+		} else if opts.Rekey.RekeyDR {
```

Comment:
Is it possible for Rekey() to return a nil err and an empty disaster key?  If it is, we should fix that.

Reply:
(same as above)